src/python/pants/module_mapping.yaml

```diff
@@ -0,0 +1,35 @@
+---
+attrs:
+  - attr
+
+enum34:
+  - enum
+
+paho_mqtt:
+  - paho
+
+pyopenssl:
+  - OpenSSL
+
+pycrypto:
+  - Crypto
+
+pymongo:
+  - bson
+  - gridfs
+
+pytest_runner:
+  - ptr
+
+python_dateutil:
+  - dateutil
+
+setuptools:
+  - easy_install
+  - pkg_resources
+
+zope.event:
```

Comment:
I left the `zope.*` packages in here, not for final, but as a discussion point on how that would work, when there are more than one package naming the same top level module.. does that work?

Reply:
I *think* we'd want to map to the highest package that is unique to the distribution. E.g., `zope.event` just maps to `zope.event` and not to `zope`. The assumption being that if you consume `zope.event` you are always importing from `zope.event` and not from `zope`. 

That seems like the most sane assumption (and means that we can omit zope here, assuming our "dist-name == top-level package name" heuristic works with dots in the name.)

Reply:
@Eric-Arellano thoughts?

Reply:
If >1 target "owns" the same module, Pants dep inference will no-op:

https://github.com/pantsbuild/pants/blob/824a15daa9f633554b5544ae2c59405601e20e1c/src/python/pants/backend/python/dependency_inference/module_mapper.py#L196-L198

This is really important that we do this, as Pants cannot disambiguate what the user intended. For example, sometimes a user will have a version of Numpy with CPU, and another with GPU. It's an error to load both at the same time. We can't disambiguate which the user wants, so we don't try to be too clever, and the user must explicitly add the dependency.

Reply:
Great. Dropped the zope entries from the mapping.